### ISSUE TITLE
Process manager doesn't update it's list of processes when the new process is created, quitted, killed by another process manager, or moved to another window

### ISSUE THREAD

**Issue by abenkovskii** [akrasuski1/akrOS#42]
I'm surprised but trying to access ghost items doesn't cause a crash and doesn't change other processes.
![screenshot3](https://cloud.githubusercontent.com/assets/11031560/7447571/a107f598-f209-11e4-8019-77658decca35.png)

**Comment by akrasuski1** [comment]
This one is a bug I knew about. I made the code such that when it doesn't find a process with PID equal to original process, it doesn't do anything, which explains your surprise. For the record, this was the very specific reason for which I introduced PIDs. Otherwise, I would just stick to index in process list. But seeing it as dynamic thing, I added constant and unique identifiers.

With current widgets I'm afraid I can't make a self-updating process manager. Also, even if I made one, I'd say user can get easily confused if the list will change without him even touching keyboard (or when he was just about to select something). One thing I can do, is to add "Refresh" button. Is that fine?
